R/print.R

```diff
@@ -48,16 +53,27 @@ print.gsplot <- function(x, ...){
     # -- call functions -- 
     to_gsplot(plots)
 
-    Axis(side=window$side[1],x=window$xlim)
-    Axis(side=window$side[2],x=window$ylim)
-
-    mtext(text=window$xlab, side=window$side[1], line = 2)
-    mtext(text=window$ylab, side=window$side[2], line = 2)
+    if("axes" %in% names(window) && window$axes){
+      Axis(side=window$side[1],x=window$xlim)
+      Axis(side=window$side[2],x=window$ylim)
+    } else {
+      Axis(side=window$side[1],x=window$xlim)
+      Axis(side=window$side[2],x=window$ylim)      
+    }
+    
+    if("ann" %in% names(window) && window$ann){
+      mtext(text=window$xlab, side=window$side[1], line = 2)
+      mtext(text=window$ylab, side=window$side[2], line = 2)
+    } else { # Get these in yaml....
+      mtext(text=window$xlab, side=window$side[1], line = 2)
+      mtext(text=window$ylab, side=window$side[2], line = 2)
+    }
     
     par(new=TRUE)
   }
-  box()
   
+  box()
+
```

Comment:
I wanted box() to be conditional with "frame.plot", but that started to seriously screw up some plots. Overplotting all over the place. I really don't see what's special about box()...but soemthing to keep in mind.